src/fontra/backends/designspace.py

```diff
@@ -1139,10 +1142,14 @@ def ufoLayerToStaticGlyph(glyphSet, glyphName, penClass=PackedPathPointPen):
     pen = penClass()
     glyphSet.readGlyph(glyphName, glyph, pen, validate=False)
     components = [*pen.components] + unpackVariableComponents(glyph.lib)
+    anchors = [structure(a, Anchor) for a in glyph.anchors]
```

Comment:
You know what, I take my advice to use "structure" here back. It's a bit of a hack, and could potentially break if UFO ever added more attributes to the anchor (unlikely, but still).
```suggestion
    anchors = [Anchor(name=a.name, x=a.x, y=a.y) for a in glyph.anchors]
```
It makes things more explicit, and show that the UFO anchor is _not_ an exact "unstructured" version of Fontra's Anchor object. As also evidenced by your findings regarding `unstructure`.